app/commands/solution/publish.rb

```diff
@@ -27,7 +27,7 @@ def award_reputation!
         solution.user,
         :published_solution,
         solution: solution,
-        level: solution.exercise.difficulty_description
+        level: solution.exercise.difficulty_category
```

Comment:
Should level be `difficulty_category` to match the model's naming? The latter is a better name than level.

Reply:
I think `level` as a keyword is quite hard coded into the reputation system so I don't think we can change it. Or at least, it's going to be more work to make it customisable per reputation type, so let's leave this one for now.